jupyter_client/manager.py

```diff
@@ -438,16 +485,16 @@ def shutdown_kernel(self, now=False, restart=False):
         # Stop monitoring for restarting while we shutdown.
         self.stop_restarter()
 
-        self.interrupt_kernel()
+        await self._async_interrupt_kernel()
```

Comment:
This breaks subclasses that override interrupt_kernel() as they won't have their override called.